sklearn/covariance/elliptic_envelope.py

```diff
@@ -68,6 +68,24 @@ class EllipticEnvelope(MinCovDet, OutlierMixin):
         such a way we obtain the expected number of outliers (samples with
         decision function < 0) in training.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.covariance import EllipticEnvelope
+    >>> from sklearn.datasets import make_gaussian_quantiles
+    >>> real_cov = np.array([[.8, .3],
+    ...                      [.3, .4]])
+    >>> np.random.seed(0)
+    >>> X = np.random.multivariate_normal(mean=[0, 0],
+    ...                                   cov=real_cov,
+    ...                                   size=500)
+    >>> cov = EllipticEnvelope(random_state=0).fit(X)
+    >>> cov.covariance_ # doctest: +ELLIPSIS
+    array([[0.7411..., 0.2535...],
+           [0.2535..., 0.3053...]])
+    >>> cov.location_
+    array([0.0813... , 0.0427...])
```

Comment:
Should we also illustrate the outlier detection interface?

Reply:
done